### PR TITLE
Add a hint on how to use `composer bin`

### PR DESCRIPTION
**Pull request by Ernestopheles**
Without this hint newbies like me ;-)) might wonder why the tools are not working...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ to a DCA of other bundles).
 
 ## Release
 
+To ernable the following run `composer update` and `composer bin all install` in your bundle's root.
+
 Run the PHP-CS-Fixer and the unit test before you release your bundle:
```
Comment (aschempp) [inline README.md]:
```suggestion
First make sure the tools are installed and up to date by running `composer bin all update` in your bundle's root.
Then run the PHP-CS-Fixer and the unit test before you release your bundle:
```

**Review by aschempp — CHANGES_REQUESTED** [review]

**Review by aschempp — APPROVED** [review]

**Review by fritzmg — APPROVED** [review]
@leofeyer this should be merged imho

File: README.md [hunk 2]
```diff
@@ -43,7 +43,8 @@ to a DCA of other bundles).
 
 ## Release
 
-Run the PHP-CS-Fixer and the unit test before you release your bundle:
+First make sure the tools are installed and up to date by running `composer bin all update` in your bundle's root.
+Then run the PHP-CS-Fixer and the unit test before you release your bundle:
```
Comment (leofeyer) [inline README.md]:
```suggestion
First make sure the tools are installed and up to date by running
`composer bin all update` in your bundle's root. Then run the PHP-CS-Fixer and
the unit test before you release your bundle:
```

**Comment by leofeyer** [comment]
Thank you @Ernestopheles.

**Pull request merged**